src/Raven.Server/Documents/Handlers/Streaming/StreamingHandler.cs

```diff
@@ -76,203 +66,21 @@ await using (var writer = new AsyncBlittableJsonTextWriter(context, ResponseBody
         [RavenAction("/databases/*/streams/queries", "HEAD", AuthorizationStatus.ValidUser, EndpointType.Read)]
         public Task SteamQueryHead()
         {
-            return Task.CompletedTask;
+            return Task.CompletedTask; //TODO stav: ?
```

Comment:
yes, same for sharding. nothing else to do here.

Reply:
done